src/Augurk.UI/Shared/MainLayout.razor

```diff
@@ -12,7 +12,7 @@
 
     <div class="main">
         <div class="top-row px-4">
-            <a href="https://augurk.github.io" target="_blank" class="ml-md-auto">About</a>
+
```

Comment:
Do we still want to place this one somewhere else?

Reply:
Maybe we should have a footer as well where we can place this link?

Reply:
Let's leave this for now and we'll revisit in a new PR.